plugins/utils_operators/agol_operators.py

```diff
@@ -12,13 +12,49 @@
 from airflow.utils.decorators import apply_defaults
 from utils import agol_utils
 
+# init some vars to be used throughout this file
+# maps agol data type to ckan data type
+AGOL_CKAN_TYPE_MAP =  {
+    "sqlTypeFloat": "float",
+    "sqlTypeNVarchar": "text",
+    "sqlTypeInteger": "int",
+    "sqlTypeOther": "text",
+    "sqlTypeTimestamp2": "timestamp",
+    "sqlTypeDouble": "float",
+    "esriFieldTypeString": "text",
+    "esriFieldTypeDate": "timestamp",
+    "esriFieldTypeInteger": "int",
+    "esriFieldTypeOID": "int",
+    "esriFieldTypeDouble": "float"
+}
+
+# list of attributes that are redundant when geometry is present in a response
+DELETE_FIELDS = [
+    "longitude",
+    "latitude",
+    "shape__length",
+    "shape__area",
+    "lat",
+    "long",
+    "lon",
+    "x",
+    "y",
+    "index_"
+]
+
 
 class AGOLDownloadFileOperator(BaseOperator):
     """
     Downloads file from AGOL URL and saves to provided directory using provided filename.
     This will always overwrite an existing file, if it's there.
     This is because we are not interested in versioning AGOL data - only pulling the latest from AGOL.
 
+    Expects the following inputs:
+        - file_url - reference to the url where this operator will grab the data
```

Comment:
I'm a little confused on this one, but I think it might be that I'm not deeply embedded in this daily.

Isn't this URL pointing to an API endpoint for that data, which returns a JSON response? Or is this an URL to an actual file?

Reply:
no youre 100% right - this is meant to be a URL from which we get a response, not a file.
Im copying the `DownloadFileOperator` language a bit too much.